gtfs/spec/en/reference.md

```diff
@@ -311,7 +313,7 @@ File: **Optional**
 
 Fare rules for individual legs of travel. 
 
-File `fare_leg_rules.txt` provides a more detailed method for modeling fare structures. As such, the use of `fare_leg_rules.txt` is entirely seperate from files `fare_attributes.txt` and `fare_rules.txt`.
+There are two modelling options for describing fares. GTFS-Fares V1 is the legacy option for describing minimal fare information. GTFS-Fares V2 is an updated method that allows for a more detailed account of an agency's fare structure. Both are allowed to be present in a dataset, but only one method should be used by a data consumer for a given dataset. It is recommended that GTFS-Fares V2 takes precedence over GTFS-Fares V1. <br><br>GTFS-Fares V1: <br>-`fare_attributes.txt`<br>-`fare_rules.txt`<br><br>GTFS-Fares V2: <br>-`fare_leg_rules.txt`<br>-`fare_transfer_rules.txt`
```

Comment:
I understand why this is copied here as well, but then whatever changes we make, we'll need to make sure in both copies it's updated. Or mabe we can move it to somewhere where we won't need it to be copied.

Reply:
I agree we should avoid duplication. Would it suffice if we only put it in the `fare_leg_rules.txt` description seeing that this is the first file in GTFS-Fares V2?

Reply:
In that case I would either opt to have it mentioned once in `fare_attributes.txt` (the "first" mention before the rest of the fares files) or I would mention it once in `fare_leg_rules.txt` (since v2 is being "added" over an old concept).

I agree this would all look cleaner formatted in a more formal way, but that's not a task I think we should concern ourselves with in the scope of this PR. However as GTFS grows and improves some sort of versioning mechanism should be put in place. See more of my thoughts on that in #288.

Reply:
Then I would either add it to fare_attributes.txt, because that's the 1st one in the file.
OR
I would move the V2 files to be before and add it there. This solution has an additional advantage: it emphasizes  that the recommendation is to produce V2.